ui/apps/dashboard/e2e/policy/overridepolicy/cluster/overridepolicy-cluster-delete.spec.ts

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2025 The Karmada Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { test } from '@playwright/test';
+import {
+    setupDashboardAuthentication,
+    generateTestClusterOverridePolicyYaml,
+    createK8sClusterOverridePolicy,
+    getClusterOverridePolicyNameFromYaml,
+    deleteOverridePolicyResourceTest
+} from './test-utils';
+
+test.beforeEach(async ({ page }) => {
+    await setupDashboardAuthentication(page);
+});
+
+test('should delete clusteroverridepolicy successfully', async ({ page }) => {
+    const testClusterOverridePolicyYaml = generateTestClusterOverridePolicyYaml();
+
+    await deleteOverridePolicyResourceTest(page, {
+        resourceType: 'clusteroverridepolicy',
+        tabName: 'Cluster level',
+        apiEndpointPattern: '/overridepolicy',  // Both namespace and cluster level use the same delete endpoint
```

Comment:
How are namespace and cluster level distinguished? This seems different from our understanding of the backend API endpoint.